meta-luneos-backports/README.md

```diff
@@ -0,0 +1,32 @@
+# Summary
+
+This layer is a collection of recipes and bbclasses we needed to backport
+from newer oe-core to our build which is using older release.
+
+The aim of this layer is to be able to just remove it from BBLAYERS when we
+upgrade oe-core instead of removing individual files from our other layers.
+
+If you're looking for backports from dylan to danny, check danny branch.
+If you're looking for backports from dora to dylan, check dylan branch.
+If you're looking for backports from daisy to dora, check dora branch.
+If you're looking for backports from dizzy to daisy, check daisy branch.
+If you're looking for backports from fido to dizzy, check dizzy branch.
+If you're looking for backports from jethro to fido, check fido branch.
+If you're looking for backports from krogoth to jethro, check jethro branch.
+If you're looking for backports from morty to krogoth, check krogoth branch.
+If you're looking for backports from pyro to morty, check morty branch.
+If you're looking for backports from rocko to pyro, check pyro branch.
+If you're looking for backports from sumo to rocko, check rocko branch.
+If you're looking for backports from thud to sumo, check sumo branch.
+If you're looking for backports from warrior to thud, check thud branch.
+If you're looking for backports from zeus to warrior, check warrior branch.
+If you're looking for backports from dunfell to zeus, check zeus branch.
+If you're looking for backports from gatesgarth, hardknott, honister, kirkstone to dunfell, check dunfell branch.
+If you're looking for backports from hardknott, honister, kirkstone to gatesgarth, check gatesgarth branch.
+If you're looking for backports from honister, kirkstone to hardknott, check hardknott branch.
+If you're looking for backports from kirkstone to honister, check honister branch.
+If you're looking for backports from langdale, mickledore, nanbield to kirkstone, check kirkstone branch.
+If you're looking for backports from mickledore, nanbield to langdale, check langdale branch.
+If you're looking for backports from nanbield to mickledore, check mickledore branch.
```

Comment:
This was true for meta-webosose in meta-webos-ports please drop this list.

Reply:
You didn't need to delete the whole file, just the list of branches which doesn't apply, but that's fine.